README.md

```diff
@@ -23,6 +23,7 @@ Optuna-Integration API reference is [here](https://optuna-integration.readthedoc
 * [ChainerMN](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#chainermn)  ([example](https://github.com/optuna/optuna-examples/tree/main/chainer/chainermn_simple.py))
 * [Keras](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#keras)  ([example](https://github.com/optuna/optuna-examples/tree/main/keras))
 * [MXNet](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#mxnet)  ([example](https://github.com/optuna/optuna-examples/tree/main/mxnet))
+* [scikit-optimize](https://optuna-integration.readthedocs.io/en/stable/reference/index.html#skopt)  ([example](
```

Comment:
It looks like the link is broken.
<img width="240" alt="image" src="https://github.com/optuna/optuna-integration/assets/30489874/16a9906d-e4ae-4994-badb-f974247829d6">

Reply:
Thanks for the comment. I removed it.